runners/flink/runner/src/main/java/org/apache/beam/runners/flink/FlinkDetachedRunnerResult.java

```diff
@@ -46,7 +45,7 @@ public State getState() {
       throws AggregatorRetrievalException {
     throw new AggregatorRetrievalException(
         "Accumulators can't be retrieved for detached Job executions.",
-        new NotImplementedException());
+        new UnsupportedOperationException());
```

Comment:
I did this so we don't need the extra commons dependency for something so simple as this.